### ISSUE TITLE
[Feature Request]: Add function to open valve

### ISSUE THREAD

**Issue by pustelbaer** [smarthausleben/ioBroker.wamo#25]
Please add a function to open the valve. The SYR app supports this. Whenever my valve is closed, I have to use the app.
Opening the valve could be a two-step process for safety reasons. Example: First the alarm must be acknowledged (boolean value). Only then would it be possible to open the valve via another data point.

**Comment by smarthausleben** [comment]
You can open the valve if you set the AB object (see picture below) to 1. Setting AB to 0 will close the valve.

![image](https://github.com/smarthausleben/ioBroker.wamo/assets/101345409/4fbc46c4-6ac1-4f66-a242-48dad4f57649)

**Comment by pustelbaer** [comment]
thank you for the info! I''ve just tested and it works like a charm. Great!

**Comment by pustelbaer** [comment]
Unfortunately, I must correct myself. The opening and closing of the valve works with AB only if the valve was closed via the adapter. If an alarm has been triggered and the SafeTech has closed the valve, it is not enough to set an AB to 1. Presumably the alarm must first be acknowledged, which is apparently not possible via the interface.

**Comment by swenp76** [comment]
> Unfortunately, I must correct myself. The opening and closing of the valve works with AB only if the valve was closed via the adapter. If an alarm has been triggered and the SafeTech has closed the valve, it is not enough to set an AB to 1. Presumably the alarm must first be acknowledged, which is apparently not possible via the interface.

I can confirm that. The alarm must be acknowledge using the "clrALA" command before the valve can be opened again.

**Comment by pustelbaer** [comment]
great. Let me know when a beta version is available. I can test the clrAla function. I do not have floor sensors in use.

**Comment by smarthausleben** [comment]
I’m always verry happy if someone else is testing the adapter. This feedback helps as well to get the adapter maybe once into the stable repository.

By the way, you can always install the development version using the “Github Cat” custom URL (see steps in the pictures below).

But be careful because this code can change anytime if I push changes to Github. So bevore doing so, a look into the Actions section may be useful. But on the other hand, if something goes wrong, you can always go back to the latest release. 

Below I will also put also a screen shot of the current action (2023.06.15 - 9:30) because this version is tested and ready for release. But since I’m waiting for a response from SYR developers, regarding Floor Sensors, it may change again soon.

**Installing current latest development version:**
![image](https://github.com/smarthausleben/ioBroker.wamo/assets/101345409/e75e5218-4ca5-4417-b5e4-ec497ad4752d)

![image](https://github.com/smarthausleben/ioBroker.wamo/assets/101345409/1fcd242c-d407-430f-8732-562038963165)

Current action status (ready for beta release as of 2023.06.15 - 09:30)
![image](https://github.com/smarthausleben/ioBroker.wamo/assets/101345409/4089392a-9e2e-4d0d-8bb7-c3c055b727af)

![image](https://github.com/smarthausleben/ioBroker.wamo/assets/101345409/511eb283-4c9c-4878-8b86-c0269271bc33)

**Comment by pustelbaer** [comment]
when I install it this way, I still get the version 0.3.0.
Do I miss something?

**Comment by smarthausleben** [comment]
No, 0.3.0 is OK because version number will change only on new releases

**Comment by pustelbaer** [comment]
unfortunately it doesn't work. If my valve is closed by an alarm state (ALM contained '[A3] Volume Leakage'), it doesn't help to set CLRALA to true. The object state will then reflect the true value, but the value stays unconfirmed. At this time the communication was not interrupted, so I could set AB to 1, but it didn't open the valve either. 
Opening the valvle via the app always works.

**Comment by smarthausleben** [comment]
Looks like the event to send the **clear alarm command** is not working like it should. I’m already investigating the problem.

**Comment by smarthausleben** [comment]
Version 0.4.0 is released. Please let me know if CLRALA works correctly.

**Comment by smarthausleben** [comment]
Version 0.4.1 is released.
Statusupdate `ALA` and `VLV` immediatly after alarm clearing (`CLRALA` command).
So this two states should be updated after a **successful** CLRALA  action.
Please let me know if CLRALA works now according to the this [Feature Request] so I can close this issue.

**Comment by pustelbaer** [comment]
with 0.4.1 it now works perfectly. Thanks again! 
The bug can be closed.

**Comment by smarthausleben** [comment]
Thanks for your feedbackv 👍

**Comment by smarthausleben** [comment]
Solved with version v0.4.1
